java/fury-core/src/main/java/org/apache/fury/logging/LoggerFactory.java

```diff
@@ -27,6 +27,7 @@
 public class LoggerFactory {
   private static volatile boolean disableLogging;
   private static volatile boolean useSlf4jlogger;
+  private static volatile int logLevel;
```

Comment:
The default value 0 will disable all fury logging

Reply:
Yes, this is a potential bug, I see you have fixed it in https://github.com/apache/incubator-fury/pull/1503, thanks!